### DISCUSSION TITLE
Huge performance difference from older may 4th version to the current ones on 6750xt

### DISCUSSION THREAD

**Discussion by Cbxxxx** [ChefKissInc/NootRX#78] [General]
Hey guys, hi! 

First, thanks for the amazing work that saved a lot of us from the community. 

I just would like to report a huge performance difference on my 6750xt 12gb from the older version of the kext to the current ones.  

I found out that all the kexts released after may 4th have a power limit of 150w, that decreases the performance a lot.

Cinebench 6 Metal scores goes from around 100k on the newer kexts to around 175k on the old one.

Tested on Sonoma 14.3.1 and sonoma 14.6, same bios configs, same everything. I've been using the older version all this time and testing once a month to see if its fixed on the newer ones.

Just reporting, 

Thanks a lot and i hope it helps!

**Comment by VisualEhrmanntraut** [comment]
Try the builds in #71
